sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamStandardJoinRel.java

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
This is an awesome PR!

Could we rename this class to BeamCoGBKJoinRel? CoGBK will show what Join approach this Rel implements.

Reply:
Done